data/text/spanish.txt

```diff
@@ -52,40 +52,40 @@ stage_complete	¡Etapa Completa!
 total			Total
 spec_taken		¿El Spec ha sido encontrado?
 stars			Estrellas
-all_stars_found ¡Has obtenido un ítem!
+all_stars_found ¡Has obtenido objeto!
```

Comment:
Should the article really be omitted here? Isn't "has obtenido **un** objeto" right?